integration_tests/test_list_07.py

```diff
@@ -1,6 +1,28 @@
 from ltypes import c64, i32
 from copy import deepcopy
 
+def generate_complex_tensors(mat: list[list[c64]], vec: list[c64]) -> list[tuple[list[list[c64]], list[c64]]]:
+    tensor: tuple[list[list[c64]], list[c64]]
+    tensors: list[tuple[list[list[c64]], list[c64]]] = []
+    rows: i32 = len(mat)
+    cols: i32 = len(vec)
+    i: i32; j: i32; k: i32
+
+    tensor = (deepcopy(mat), deepcopy(vec))
+
+    for k in range(2 * rows):
+        tensors.append(deepcopy(tensor))
+        for i in range(rows):
+            for j in range(cols):
+                mat[i][j] += complex(1.0, 2.0)
+
+        for i in range(cols):
+            vec[i] += complex(1.0, 2.0)
+
+        tensor = (deepcopy(mat), deepcopy(vec))
+
+    return tensors
```

Comment:
(I personally do not like using the term tensor to mean multidimensional array, but we can change that later.)

Reply:
Sure.